### ISSUE TITLE
PEER Review setup does not show the preview for ratings for 5 Star Rating with criteria or without out criteria

### ISSUE THREAD

**Issue by mzwolinski** [ecsion/eSeaGrant-Client-Bug-Tracker#661]
PEER Review setup does not show the preview for ratings for 5 Star Rating with criteria or without out criteria.
http://eseagrant.texasseagrant.org/index.php?module=RFP&view=Detail&mode=showPeer&record=129&phaseid=184&tab_label=Full-Proposals

You could see the preview when setting up the Panel Review in the Pre-proposals, so that you could see what it would look like.
http://eseagrant.texasseagrant.org/index.php?module=RFP&view=Detail&mode=showPanelReviewerList&record=129&phaseid=184&tab_label=Full-Proposals&listtype=invitedpanel

Also, is there a way to change the criteria from 3 boxes to only 2 criteria boxes?

Thank you!
[Panel Review setup without Criteria preview popup.pdf](https://github.com/ecsion/eSeaGrant-Client-Bug-Tracker/files/906175/Panel.Review.setup.without.Criteria.preview.popup.pdf)
[Peer Review setup with Criteria preview popup.pdf](https://github.com/ecsion/eSeaGrant-Client-Bug-Tracker/files/906176/Peer.Review.setup.with.Criteria.preview.popup.pdf)

Mia

**Comment by ecsion** [comment]
We will add this. It will take us some time to do this. We will let you know once it is done.

**Comment by ecsion** [comment]
fixed. Please check.

**Comment by ecsion** [comment]
Mia, we are closing this issue. Please reopen if it is not fixed.

**Comment by mzwolinski** [comment]
Hi Deepak and Ecsion Team,

Thank you for providing the fix 2 days ago for my request of 12 days ago; I tested and it was successful.

As you know, Texas Sea Grant was ahead of schedule for the Biennial Research Competition as compared to other Sea Grant Programs, with Full Proposals due April 3rd and our internal deliverable to release the proposals to the Peer Reviewers on April 10th. Unfortunately, we had to make the decision not to utilize eSeaGrant for this purpose as we literally ran out of time and sent the 27 proposals to the 48 Peer Reviewers manually.

Perhaps there may be a way to still utilize the Panel Review module for the Technical Panel Review process expected to start mid-May. I know that there have been good suggestions to improve that module for the anticipated future release of eSeaGrant version 2.0, however, I am unsure of what has been fixed so far and what will be instead included in version 2.0. I may need to seek yours and Kathy’s assistance and recommendations as to how to incorporate our manual Peer review results into the Panel module in order to have an understanding of whether to use eSeaGrant for our mid-May timeframe.

A very time-consuming task was related to compiling the full proposal package by the Administrator, when this should have been the responsibility of the PI. The practicality of having separate components, downloading the zipfile of all of the components, only to have to then sort through which documents were a true proposal document component, with varied document filenames and compile a proposal package. Additionally, component documents were included in the zipfile, which were not the specific proposal component document; rather it was a document which then contained a link to open the proposal component document. Having to do this for 35 proposals was quite time-consuming.

Upon reflection, I should have created a separate component as I did for the Pre-proposal submission, for the PIs to simply upload one entire PDF. While considering that at the time, I was perplexed while trying to develop the full proposal module, while at the same time Ecsion was making fixes. It appeared that I was having the PIs waste their time inputting information and uploading separate component documents, but as I am not a software engineer, kept thinking that there was a good reason for having the separate components and had the PIs proceed with that method.

Another discouraging result related to the PIs having to enter Project Summary and Data Management information into text boxes, which fed the new Project Summary page, but for which was not accessible to the PIs as a download for them to view. Additionally, the text out-put on the Project Summary page was quite poor, very small font, very light and had narrow and offset margins.

I know that you all are gathering information to summarize fixes, new requests, planned upgrade features for version 2.0 and that we will have a webinar/conference call soon. We appreciate your efforts.

Thank you and regards,
Mia

Mia Zwolinski, Research Coordinator
Texas Sea Grant College Program | Texas A&M University | 979-458-0449
http://texasseagrant.org/
[cid:501AB10A-6550-423C-967F-9827F6D0E23F]
